crates/runtime/src/accelerated_table/refresh.rs

```diff
@@ -130,8 +131,13 @@ impl Refresher {
 
             match future_result {
                 Some(result) => {
-                    let Ok((start_time, data_update)) = result else {
-                        continue;
+                    let (start_time, data_update) = match result {
+                        Ok((start_time, data_update)) => (start_time, data_update),
+                        Err(e) => {
+                            tracing::debug!("Error getting update for dataset {dataset_name}: {e}");
```

Comment:
Writing as debug instead of error to not duplicate the error (there is already error from connection stream)